test/test_multi_region_cubed_sphere.jl

```diff
@@ -224,7 +259,7 @@ end
 
             Nx, Ny, Nz = 9, 9, 1
 
-            grid = ConformalCubedSphereGrid(arch, FT; panel_size = (Nx, Ny, Nz), z = (0, 1), radius = 1, horizontal_direction_halo=2)
+            grid = ConformalCubedSphereGrid(arch, FT; panel_size = (Nx, Ny, Nz), z = (0, 1), radius = 1, horizontal_direction_halo = 3)
```

Comment:
any reason for changing the halo to 3? (just wondering..)

Reply:
OK, after talking with @siddharthabishnu in person seems that there is no specific reason.

Reply:
Exactly. Just wanted the number of halos to be consistent for fields at locations at cca, fca, cfa and ffa. Moreover, the existing tests passed with 3 halos. So, we are good.

Reply:
but `horizontal_direction_halo = 2` is also consistent across cca, fca, cfa and ffa fields, no?

Reply:
Yes, it is. I shouldn't have said 'consistent'. I meant I wanted to specify the same number of halos for testing both the tracer and velocity fields. It doesn't change the end result.

Reply:
Gotcha